src/main/java/edu/cornell/kfs/concur/rest/jsonObjects/ConcurV4WorkflowDTO.java

```diff
@@ -0,0 +1,38 @@
+package edu.cornell.kfs.concur.rest.jsonObjects;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/*
+ * NOTE: This DTO has been limited to just the common field(s) for both Expense V4 and Request V4 workflow actions.
+ * If further separation is needed between the two in the future, then please refactor this class accordingly.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ConcurV4WorkflowDTO {
+
+    @JsonProperty("comment")
+    private String comment;
+
+    public ConcurV4WorkflowDTO() {
+    }
```

Comment:
Is an explicit default constructor required?

Reply:
The Jackson framework might need a default constructor in order to build this DTO from a JSON value. Even though this PR's new Production code is only doing DTO-to-JSON conversion for this clas, the unit test's MVC controller is implicitly relying on Jackson's JSON-to-DTO conversion for the class.